### ISSUE TITLE
Ability to read Variables from .env files

### ISSUE THREAD

**Issue by rafilkmp3** [fabfuel/ecs-deploy#32]
I am using your amazing tool with jenkins to make my CD, but I'd like to be able to provide variables from 
 an environment file where I have it tracked which git, can you in a future release a way to support this ?

thanks again for your effort and bring us this amazing tool :)

**Comment by brunocascio** [comment]
+1

**Comment by stadskle** [comment]
@fabfuel interesting in a PR on this? We would possible consider this to simplify our flow. Thoughts on syntax?

Something like this?
```phyton
@click.option('--env-file', required=False, help='load environment variables from file')
```

**Comment by fabfuel** [comment]
Yes, that would be awesome and highly appreciated @stadskle!
Syntax looks good 👍 

Best
Fabian

**Comment by stadskle** [comment]
> Yes, that would be awesome and highly appreciated @stadskle!
> Syntax looks good 👍

Great, I have found someone that will take a look at it. Hoping to have a PR to you next week

**Comment by stadskle** [comment]
@fabfuel we have done some tests on this in our environment and it seems to be working fine, great if you can have a look soon and make a release, se we can git it into our pip install flow

**Comment by fabfuel** [comment]
Thanks @stadskle, I will have a look on it soon!

**Comment by fabfuel** [comment]
This has been solved in #150, i will publish a new version soon. Thanks everyone!
